src/app/profile/page.js

```diff
@@ -13,8 +14,9 @@ export default function Profile() {
   };
   return (
     <main>
-      <div>
-        <p>I am the Profiles page</p>
+      <Navbar />
+      <div className="container">
```

Comment:
Usually I don't just use the name container for a big div, I try to specify it as profile-page-container for example. I think it is easier to overview later. You don't have to change that, just to let you know :)